system/repositories.rs

```diff
@@ -153,9 +155,9 @@ pub mod user_repository {
         let unexpected_error_msg = "An unexpected error occurred when attempting to delete a user";
         let query = parse_query(
             format!(
-                "match $up isa user-password, links (user: $u, password: $p);
+                "match $uc isa user-credentials, links (user: $u, credentials: $p);
```

Comment:
`credentials: $p` -> `credentials: $c` would be a little cleaner
with `delete $u; $p; $uc;",` -> `delete $u; $c; $uc;",`